pkg/servergroup/config.go

```diff
@@ -104,7 +104,7 @@ type Config struct {
 	AntiAffinity time.Duration `yaml:"anti_affinity,omitempty"`
 
 	// Timeout, if non-zero, specifies the amount of
-	// time to wait for a server's response headers after fully
+	// time in seconds to wait for a server's response headers after fully
```

Comment:
We actually don't want to say in seconds as this is a `time.Duration` so it'll properly parse durations such as `1s` or `5m` -- so IMO this second commit is unnecessary (and doesn't match the other duration stuff such as anti_affinity above).

Reply:
Thanks for your feedback on this and help to push this forward :)

Reply:
I rebased from upstream/master